zabbix/README.adoc

```diff
@@ -0,0 +1,95 @@
+Plugin for ZABBIX and Rudder
+----------------------------
+
+This project is part of Rudder - IT automation and compliance made easy.
+See: http://rudder-project.org for more information.
+
+Mailing list, irc : 
+https://www.rudder-project.org/site/community/mailing-lists/
+
+Synposis
+~~~~~~~~
+
+This plugin aims at providing Rudder integration with Zabbix. It can automatically
+add hosts to Zabbix when the corresponding node is set up for monitoring in Rudder.
+
+The nodes can also have Zabbix macros applied using the appropriate generic method directly in
+Rudder's technique editor.
+
+// Everything after this line goes into Rudder documentation
+// ====doc====
+
+= Zabbix
+
+This plugin aims at providing Rudder integration with Zabbix. It can automatically add hosts to Zabbix
+when the corresponding node is set up for monitoring in Rudder
+
+
+== Installation
+
+
+* Prerequisites :
+
+  - Install a Zabbix server https://www.zabbix.com/download. The server is not required to run on the same host as the Rudder server.
+    However, this plugin has to be installed and running on the Rudder server.
+
+  - The Rudder server must have Python installed.
+
+* Generate the Rudder package if you do not already have the .rpkg file :
```

Comment:
This build doc should not be included in the manual either (we suppose users will download pre-built plugins and directly install and use them).